environment-dev.yaml

```diff
@@ -0,0 +1,38 @@
+name: ragna-dev
+channels:
+  - conda-forge
+dependencies:
+  - python =3.9
+  - redis-server
+  - pip
+  - aiofiles
+  - emoji
+
+  # for development only
+  - pre-commit
+
+  # For use with Panel branch for Chat Interface
+  - nodejs
```

Comment:
I couldn't find any usage of `node`. What was this for? Can we remove this whole file?

Reply:
I guess my comment isn't clear enough, sorry about that. You need nodejs to install a panel version from a git repo, panel needs nodejs at some point in its installation process. This can be removed once the next Panel version is released

Reply:
In the light of https://github.com/Quansight/ragna/pull/61#discussion_r1367989031, we can probably just drop the file.